python/pyspark/sql/connect/plan.py

```diff
@@ -1950,6 +1950,46 @@ def plan(self, session: "SparkConnectClient") -> proto.Relation:
         return plan
 
 
+class CoGroupMap(LogicalPlan):
+    """Logical plan object for a CoGroup Map API: applyInPandas."""
+
+    def __init__(
+        self,
+        input: Optional["LogicalPlan"],
+        input_grouping_cols: Sequence[Column],
+        other: Optional["LogicalPlan"],
+        other_grouping_cols: Sequence[Column],
+        function: "UserDefinedFunction",
+        cols: List[Column],
```

Comment:
@xinrong-meng just realised that we don't use this variable. Should we remove? or should it be assigned to somewhere?